apps/server/src/modules/learnroom/uc/course-copy.uc.ts

```diff
@@ -56,7 +56,7 @@ export class CourseCopyUC {
 			const boardCopy = statusBoard.copyEntity as Board;
 			await this.boardRepo.save(boardCopy);
 			statusBoard = this.lessonCopyService.updateCopiedEmbeddedTasks(statusBoard);
-			statusBoard = await this.fileCopyAppendService.copyFiles(statusBoard, courseCopy.id, userId, jwt);
+			statusBoard = await this.fileCopyAppendService.copyFiles(statusBoard, courseCopy.id, userId, user.school.id, jwt);
```

Comment:
You remove the schoolId (mapper) in file storage client, but add this on this place. I think it make more sense to hold it on the files. Because it is possible to copy files from one school to a other school. (shared links). 

I think the switch will not work.